tests/conftest.py

```diff
@@ -3,8 +3,15 @@
 
 from ibex_bluesky_core.run_engine import get_run_engine
 
+MOCK_PREFIX = "UNITTEST:MOCK:"
+
 
 @pytest.fixture
 def RE() -> RunEngine:
     get_run_engine.cache_clear()
-    return get_run_engine()
+    re = get_run_engine()
```

Comment:
purely convention, and I know it flies in the face of PEP8, but bluesky uses `RE` for the runengine everywhere rather than `re`...